x-pack/test/functional/apps/code/repo_archiver.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import extractZip from 'extract-zip';
+import fs from 'fs';
+import mkdirp from 'mkdirp';
+import path from 'path';
+import rimraf from 'rimraf';
+
+const archiveFilePath = (zipFileName: string) => {
+  return path.resolve(__dirname, `./fixtures/${zipFileName}.zip`);
+};
+
+const repoDir = (repoUri: string, kibanaDir: string) => {
+  return `${kibanaDir}/data/code/repos/${repoUri}`;
```

Comment:
Please use `path.resolve()` here and in `workspaceDir()`

Reply:
done